### PR TITLE
Create special offers slider

### PR DESCRIPTION
**Pull request by AhmedAtef277**

### CODE REVIEW

File: e-store/e-store/ScenesScreen/Home/SpecialOffersCollectionView/SpecialOffersSlider.swift [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+    private func commonInit() {
+        fromNib(type: SpecialOffersSlider.self)
+    }
+    private func registerCell() {
+        specialOffersCollectionView.register(UINib(nibName: SpecialOffersCollectionViewCell.identifier, bundle: nil)
+                                             , forCellWithReuseIdentifier: SpecialOffersCollectionViewCell.identifier)
+    }
+    
+    func setupDataSourceList() {
+        dataSource = [SpecialOffersModel(productImage: UIImage(named: Asset.airpodsRemovebgPreview.name) ?? UIImage(),
```
Comment (Mahmoud-Bassuni) [inline e-store/e-store/ScenesScreen/Home/SpecialOffersCollectionView/SpecialOffersSlider.swift:49]: plz move those mock into view model

Comment (AhmedAtef277) [reply e-store/e-store/ScenesScreen/Home/SpecialOffersCollectionView/SpecialOffersSlider.swift:49]: Mock has images so i will need to import UIKit in view model

**Review by hassanSayed0 — APPROVED** [review]

**Review by ZeyadSultan — APPROVED** [review]
Great Job

**Review by Mahmoud-Bassuni — APPROVED** [review]

**Review by hassanSayed0 — APPROVED** [review]

**Review by ZeyadSultan — APPROVED** [review]

**Review by Mahmoud-Bassuni — COMMENTED** [review]
🌹

**Pull request merged**